modules/network-observability-roles-create.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-roles-create_{context}"]
+
+= Create roles for authentication and authorization
+Specify authentication and authorization configurations by defining `ClusterRole` and `ClusterRoleBinding`. You can create a YAML to define these roles.
+.Procedure
+
+. Using the web console, navigate to the Imoport icon *+*, in the top right corner. 
+. Drop your YAML file into the editor and click *Create*: 
++
+[source, yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: loki-netobserv-tenant
+rules:
+- apiGroups:
+  - 'loki.grafana.com'
+  resources:
+  - network
+  resourceNames:
+  - logs
+  verbs:
+  - 'get'
+  - 'create'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: loki-netobserv-tenant
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: loki-netobserv-tenant
+subjects:
+- kind: ServiceAccount
+  name: flowlogs-pipeline-transformer  <1>
```

Comment:
The ServiceAccount name to use here differs if we are using Kafka or not. If not using Kafka, it would be named `flowlogs-pipeline`, so it should be mentioned.

I would show the no-kafka scenario here (hence  named `flowlogs-pipeline`), and mention in the note <1> to set it to `flowlogs-pipeline-transformer` when `FlowCollector` `spec.deploymentModel` is `KAFKA`.

Reply:
AH that's a devilish detail! Will add it.